scripts/sphinx_extensions/options_from_kconfig/options_from_kconfig.py

```diff
@@ -84,17 +84,26 @@ def run(self):
                    (prefix.startswith("'") and prefix.endswith("'")):
                     prefix = prefix[1:-1]
                 text += prefix
-            help_ = f'{sym.nodes[0].prompt[0]}'
+            try:
+                prompt_ = f'{sym.nodes[0].prompt[0]}'
+            except Exception:
+                prompt_ = ''
             if prefix is not None:
-                text += help_[:1].lower() + help_[1:]
+                text += prompt_[:1].lower() + prompt_[1:]
             else:
-                text += help_
+                text += prompt_
             if suffix is not None:
                 if (suffix.startswith('"') and suffix.endswith('"')) or \
                    (suffix.startswith("'") and suffix.endswith("'")):
                     suffix = suffix[1:-1]
                 text += suffix
             lines.append(f'{text}\n')
+            try:
+                help_ = sym.nodes[0].help
+            except Exception:
+                help_ = None
+            if help_ is not None:
+                lines.append(f'{help_}\n')
```

Comment:
You could just add the `lines.append(f'{help_}\n')` inside the `try` block and use a `pass` in the `except` block and remove this `if` block.

Reply:
changed